Makefile

```diff
@@ -139,6 +139,10 @@ manifests: ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 	$(KUSTOMIZE) build config/default > config/render/capm3.yaml
 
+.PHONY: crd
+crd: ## Generate CRD
```

Comment:
Unlike `make manifests` this is not running controller-gen. I guess that's OK if we have a CI job running generate-check to make sure the correct CRD is always checked in. It doesn't appear that we do though.

Reply:
I had missed that. Updating make crd to include a call to controller-gen to generate the CRD.

Reply:
Also based on @andfasano 's earlier comment, i 'll propose an upstream patch to update the Makefile.

Reply:
Localized all changes to Dockerfile.ocp so that we do not have to modify the upstream Makefile.

Reply:
Good idea, it seems a better approach, as it keeps all the related changes in one place only (and downstream related!)